src/styles/index.css

```diff
@@ -0,0 +1,62 @@
+#app {
+  font-family: Montserrat, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-size: 16px;
+  color: #282828;
+}
+
+.container {
+  width: 100%;
+  margin: 0 auto;
+  padding: 30px 0px;
+}
+@media screen and (min-width: 480px) {
+  .container {
+    width: 480px;
+  }
+}
+@media screen and (min-width: 768px) {
+  .container {
+    width: 768px;
+  }
+}
+@media screen and (min-width: 1200px) {
+  .container {
+    width: 1200px;
+  }
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+button {
+  display: block;
```

Comment:
Buttons, by default, do not occupy the entire row like block elements do. Therefore, using display: inline-flex; would be a more appropriate value to maintain the button's inline nature while still applying flexbox properties.